spec/build_spec_helper.cr

```diff
@@ -15,15 +15,20 @@ module Amber::CLI
     options = ["user:reference", "name:string", "body:text", "age:integer", "published:bool"]
     temp_options = options - ["user:reference", "age:integer"]
     MainCommand.run ["generate", "auth", "User"] | (options - ["user:reference"])
-    MainCommand.run ["generate", "error"]
+    MainCommand.run ["generate", "error", "ErrorPage"]
```

Comment:
this was the line making the test suite not execute

Reply:
Thanks. Possibly it should just be commented out with a TODO?

Reply:
I thought it was worth to test if the error pages are working (maybe the original intent here) that's why I added the name "ErrorPage"

Reply:
it was not executing because ```amber generate error``` just shows the help message since it's missing the error page name